index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
-      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap&family=Inter&display=swap"
```

Comment:
- [ ] [OPTIONAL] Good job 👍👏. Please kindly add a screenshot of your website to the readme file. 
-[ ] [OPTIONAL] The live demo link is not working also don't forget to fix this issue as well 🍀👋